collect/stressRelief.go

```diff
@@ -119,25 +179,62 @@ func (s *StressRelief) Start() error {
 	return nil
 }
 
-func (s *StressRelief) UpdateFromConfig(cfg config.StressReliefConfig) error {
+type stressReliefMessage struct {
+	level  uint
+	peerID string
+}
+
+func newStressReliefMessage(level uint, peerID string) *stressReliefMessage {
+	return &stressReliefMessage{level: level, peerID: peerID}
+}
+
+func (msg *stressReliefMessage) String() string {
+	return msg.peerID + ":" + fmt.Sprint(msg.level)
+}
+
+func unmarshalStressRelifMessage(msg string) (*stressReliefMessage, error) {
+	if len(msg) < 2 {
+		return nil, fmt.Errorf("empty message")
+	}
+
+	parts := strings.SplitAfter(msg, ":")
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("invalid message format")
+	}
+
+	level, err := strconv.Atoi(parts[1])
+	if err != nil {
+		return nil, err
+	}
+
+	return newStressReliefMessage(uint(level), parts[0]), nil
+}
+
+func (s *StressRelief) onStressLevelUpdate(ctx context.Context, msg string) {
+	stressMsg, err := unmarshalStressRelifMessage(msg)
+	if err != nil {
+		s.Logger.Error().Logf("failed to unmarshal stress relief message: %s", err)
+		return
+	}
+
+	s.lock.Lock()
+	defer s.lock.Unlock()
+
+	s.stressLevels[stressMsg.peerID] = stressReport{
+		key:       stressMsg.peerID,
+		level:     stressMsg.level,
+		timestamp: s.Clock.Now(),
+	}
+}
+
+func (s *StressRelief) UpdateFromConfig(cfg config.StressReliefConfig) {
 	s.lock.Lock()
 	defer s.lock.Unlock()
 
 	switch cfg.Mode {
 	case "never", "":
 		s.mode = Never
 	case "monitor":
-		// If we're switching into monitor mode from some other state (which
-		// happens on startup), we will start up in stressed mode for a
-		// configurable time to try to make sure that we can handle the load
-		// before we start processing it in earnest. This is to help address the
-		// problem of trying to bring a new node into an already-overloaded
-		// cluster. If the time is 0 we won't do this at all.
-		if s.mode != Monitor && cfg.MinimumStartupDuration != 0 {
```

Comment:
Since we're no longer using MinimumStartupDuration, we should obsolete it in the docs and remove it from the test code.